cmd/cloud/server.go

```diff
@@ -44,6 +44,7 @@ func init() {
 	serverCmd.PersistentFlags().Bool("keep-database-data", true, "Whether to preserve database data after installation deletion or not.")
 	serverCmd.PersistentFlags().Bool("keep-filestore-data", true, "Whether to preserve filestore data after installation deletion or not.")
 	serverCmd.PersistentFlags().Bool("debug", false, "Whether to output debug logs.")
+	serverCmd.PersistentFlags().Bool("machine-readable-logs", true, "Output the logs in machine readable format. Default to true.")
```

Comment:
I am pretty sure we agreed yesterday that not changing the default log behavior would be best. Shouldn't this be false for the default?

Reply:
i think so 😇 
but in terms of deployment and operation, this is the best option, fewer changes in the environments.
but if everybody wants to have as false I can make it.

Reply:
set to false